.github/workflows/Build.yml

```diff
@@ -0,0 +1,84 @@
+name: Python 3.7 Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        # os: [macos-10.15, ubuntu-18.04, windows-2019]
+        os: [macos-10.15, ubuntu-18.04]
+        include:
+          - os: ubuntu-18.04
+            cmake-build-type: "Release"
+#           - os: windows-2019
+#             cmake-build-type: "Release"
+          - os: macos-10.15
+            cmake-build-type: "Release"
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install build dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install ninja
+        python -m pip install setuptools
+        python -m pip install scikit-ci-addons
+        python -m pip install wheel
+
+    - name: Get specific version of CMake, Ninja
+      uses: lukka/get-cmake@v3.18.3
+
+    - name: Build SimpleITK Unix
+      if: matrix.os != 'windows-2019'
+      run: |
+        export CXXFLAGS="$CXXFLAGS -fPIC"
+        mkdir SIb
+        cd SIb
+        cmake -DWRAP_LUA:BOOL=OFF -DWRAP_PYTHON:BOOL=ON -DWRAP_JAVA:BOOL=OFF -DWRAP_CSHARP:BOOL=OFF -DWRAP_TCL:BOOL=OFF -DWRAP_R:BOOL=OFF -DWRAP_RUBY:BOOL=OFF -DBUILD_EXAMPLES:BOOL=OFF -DBUILD_TESTING:BOOL=ON -DCMAKE_BUILD_TYPE=${{ matrix.cmake-build-type }} -DSimpleITK_USE_SYSTEM_ITK:BOOL=OFF -v -GNinja ../SuperBuild
+        ninja
+
+    - name: Build SimpleITK Windows
+      if: matrix.os == 'windows-2019'
+      shell: cmd
+      run: |
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+        mkdir SIb
+        cd SIb
+        cmake -DWRAP_LUA:BOOL=OFF -DWRAP_PYTHON:BOOL=ON -DWRAP_JAVA:BOOL=OFF -DWRAP_CSHARP:BOOL=OFF -DWRAP_TCL:BOOL=OFF -DWRAP_R:BOOL=OFF -DWRAP_RUBY:BOOL=OFF -DBUILD_EXAMPLES:BOOL=OFF -DBUILD_TESTING:BOOL=ON -DCMAKE_BUILD_TYPE=${{ matrix.cmake-build-type }} -DSimpleITK_USE_SYSTEM_ITK:BOOL=OFF -v -GNinja ../SuperBuild
+        ninja
+
+    - name: Test SimpleITK
+      run: |
+        cd SIb/SimpleITK-build
+        ctest -C Release
+
+    - name: Build Wheel
+      run: |
+        cd SIb/SimpleITK-build/Wrapping/Python
+        python setup.py bdist_wheel
+
+    - name: Prepare Artifacts
+      shell: bash
```

Comment:
@ViktorvdValk  This and the next step prepare and upload artifacts (publishing stuff), they are not needed for the initial steps of get C++ building and testing CI working.